### PR TITLE
Docker compose

### PR DESCRIPTION
**Pull request by kapdon**
docker compose for release n dev for easy deployment.

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+services:
+  tarkov-sitcoop-dev:
+    build:
+      context: .
+      args:
+        SIT_BRANCH: "master"
+        SIT: "26b9c364963ba49de71d5761bed1135ddad50f77"
```
Comment (devbence) [inline docker-compose-dev.yml]: dev branch should be development
dev commit should be HEAD^ (grab latest)

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        SPT_BRANCH: "3.8.1-DEV"
+        SPT: "a425b1bc4bfe5403b957132dbd678afe4b9f7a21"
```
Comment (devbence) [inline docker-compose-dev.yml]: same as above, HEAD^ should be fine

File: docker-compose-dev.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    container_name: sitcoop-dev
+    environment:
+      PUID: 1000
```
Comment (devbence) [inline docker-compose-dev.yml]: what if user is not 1000?

**Comment by kapdon** [comment]
phew, ready for review, ignore all the stupidity.

File: docker-compose-dev.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+services:
+  tarkov-sitcoop-dev:
+    build:
+      context: .
+      args:
+        SIT_BRANCH: "development"
+        SIT: "HEAD^"
+        SPT_BRANCH: "3.9.0-DEV"
```
Comment (kapdon) [inline docker-compose-dev.yml]: development looks like it's on 3.9.0 lmk if other wise.

**Comment by kapdon** [comment]
wait it doesn't actually work by default, while investigating websocket stuff i thought i had fucked something up then I reset it all to master and built using HEAD^ 3.8.1-DEV or 3.9.0-DEV and it refuses to load SITCoop, giving some error about not able to get depedencies even if autoinstall dependencies is turned on in Aki, I see we recently nuked the node_modules folder I wonder if that's what's causing it, but I don't really know nodejs enough to debug.

File: docker-compose-dev.yml (hunk 4 above)
Comment (devbence) [reply docker-compose-dev.yml]: for now its 3.8.1-DEV
(the next scope will be 3.9.0-DEV (29862)) _((probably))(((maybe)))_

**Comment by devbence** [comment]
> wait it doesn't actually work by default, while investigating websocket stuff i thought i had fucked something up then I reset it all to master and built using HEAD^ 3.8.1-DEV or 3.9.0-DEV and it refuses to load SITCoop, giving some error about not able to get depedencies even if autoinstall dependencies is turned on in Aki, I see we recently nuked the node_modules folder I wonder if that's what's causing it, but I don't really know nodejs enough to debug.

#12 can you give it a try using this pr?

**Comment by kapdon** [comment]
resolved pending

**Review by devbence — APPROVED** [review]

**Pull request merged**